### DISCUSSION TITLE
Bounding boxes size does not match the ground truth as displayed on QGIS

### DISCUSSION THREAD

**Discussion by AashishKumbashi** [azavea/raster-vision#1983] [Q&A]
This might be something stupid. I was finally able to get an objection detection task to train using rastervision. But I see something strange with the ground truth bounding boxes. My input .geojson is in EPSG:25832. However, when I use a CRS transformer and plot the dataset using rastervision, I see that the labels are all perfectly aligned. My challenge is with the size of the bounding box. The size of the bounding box displayed when the data is loaded into rastervision is slightly bigger than what I can see on QGIS. I am not very sure on what I am missing here.
The blue boxes are ground truth as seen on QGIS while the black one is what I get when I load in rastervision.
![Screenshot from 2023-11-07 10-51-52](https://github.com/azavea/raster-vision/assets/104180642/68a4f9f8-0417-4300-ba53-2dfba242fd3c)

**Comment by AdeelH** [comment]
Raster Vision currently only supports EPSG:4326 GeoJSONs. You will need to convert your GeoJSONs to that before feeding them into Raster Vision.

Edit 2024-04-11: **this is no longer true in v0.30. RV now supports GeoJSONs with arbitrary CRS's.**

**Reply by AashishKumbashi** [reply]
Thanks for your response!
My bad, this issue popped up because of a bug in my preprocessing code. However, I still ended up using data in EPSG:25832 with RasterioCRSTransfomers and it seems to work well. The predictions in the end will be in EPSG:4326 but otherwise, it worked well!
